process/economics/apiEconomicsData_test.go

```diff
@@ -0,0 +1,76 @@
+package economics_test
+
+import (
+	"fmt"
+	"math/big"
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-go-core/core/check"
+	"github.com/ElrondNetwork/elrond-go-core/data/transaction"
+	"github.com/ElrondNetwork/elrond-go/process"
+	"github.com/ElrondNetwork/elrond-go/process/economics"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewAPIEconomicsData(t *testing.T) {
+	t.Parallel()
```

Comment:
`t.Parallel` on each inner test, as right now the inner tests will be run serial

Reply:
please check. Can not be done.